nodejs/awsx/ec2/vpc.ts

```diff
@@ -71,12 +73,12 @@ export class Vpc extends pulumi.ComponentResource {
                 enableDnsHostnames: utils.ifUndefined(args.enableDnsHostnames, true),
                 enableDnsSupport: utils.ifUndefined(args.enableDnsSupport, true),
                 instanceTenancy: utils.ifUndefined(args.instanceTenancy, "default"),
-            });
+            }, parentOpts);
             this.id = this.vpc.id;
 
             // Create the appropriate subnets.  Default to a single public and private subnet for each
             // availability zone if none were specified.
-            const topology = new VpcTopology(this, name, cidrBlock, numberOfAvailabilityZones, opts);
+            const topology = new VpcTopology(this, name, cidrBlock, numberOfAvailabilityZones);
```

Comment:
opts was not desirable here.  We don't want to pass along opts passed into us to child resource.  We just want to pass 'this' along which can be used as hte parent of all children.